bin/startNextflowDragenPipeline.sh

```diff
@@ -275,10 +275,20 @@ else
 		
 			module load nextflow
 			thisDir=$(pwd)
-
+			
 			cd "${TMP_ROOT_DIR}/nextflow/${project}"
-			nextflow run --samplesheet "${sampleSheet}" --tmpdir "${TMP_LFS}" --group "${group}" -w "${TMP_ROOT_DIR}/nextflow/${project}" "${EBROOTNF_NGS_DNA}/main.nf"
-
+			type=$(echo "${project#*-}" | awk 'BEGIN {FS="_"}{print $1}')
+			if [[ "${type}" == "sWGS" ]]
+			then
+				log4Bash 'TRACE' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "type=[${type}] --> sWGS project detected."
+				nextflow run --samplesheet "${sampleSheet}" --tmpdir "${TMP_LFS}" --group "${group}" -w "${TMP_ROOT_DIR}/nextflow/${project}" "${EBROOTNF_NGS_DNA}/swgs.nf"
```

Comment:
When you change swgs.nf in sWGS.nf and wgs.nf in WGS.nf, you can take the almost redundant nextflow command out of the if then else loop and use
```nextflow run --samplesheet "${sampleSheet}" --tmpdir "${TMP_LFS}" --group "${group}" -w "${TMP_ROOT_DIR}/nextflow/${project}" "${EBROOTNF_NGS_DNA}/${type}.nf"```
after the loop. The alternative is to not change the name of the workflow files and always lowercase the type with ```,,``` like this:
```nextflow run --samplesheet "${sampleSheet}" --tmpdir "${TMP_LFS}" --group "${group}" -w "${TMP_ROOT_DIR}/nextflow/${project}" "${EBROOTNF_NGS_DNA}/${type,,}.nf"```